hal/x86_64_efi.c

```diff
@@ -0,0 +1,257 @@
+#ifdef PLATFORM_X86_64_EFI
```

Comment:
This needs to be at line 30. Not everyone will define the platform in the CFLAGS. We should support getting them from the wolfBoot headers...